examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java

```diff
@@ -51,7 +51,7 @@
  *
  * <p>Basic concepts, also in the MinimalWordCount, WordCount, and DebuggingWordCount examples:
  * Reading text files; counting a PCollection; writing to GCS; executing a Pipeline both locally
- * and using a selected runner; defining DoFns; creating a custom aggregator;
```

Comment:
It has not been there for a while (at least since last July https://github.com/pabloem/incubator-beam/blob/ca9e337203208c7c5876f0710fb3a45430a5b3a8/examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java)